src/components/autocomplete/autocomplete.spec.js

```diff
@@ -1154,4 +1154,66 @@ describe('<md-autocomplete>', function() {
     }));
   });
 
+  describe('required', function () {
```

Comment:
Please notice, that we already have a section for `required`.

https://github.com/angular/material/blob/master/src/components/autocomplete/autocomplete.spec.js#L1035